examples/jupyter/integrations/NLTK.ipynb

```diff
@@ -29,96 +29,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
```

Comment:
Why did you clear the output for jupyter notebook?

Reply:
We could only remove the concrete cell output but in a single cell there is other information about initialization like.

```bash
2023-04-06 12:15:19,701	INFO worker.py:1553 -- Started a local Ray instance.

UserWarning: Distributing <class 'pandas.core.frame.DataFrame'> object. This may take some time.
```

And this also looks to me a little inconsistent when some cells are executed but other aren't. I would leave all cells not executed.

Reply:
I think as long as Modin produces this output, we can leave it there. However, we should also consider not showing such low-level information, at least by default.

> And this also looks to me a little inconsistent when some cells are executed but other aren't. I would leave all cells not executed.

Why? Is it unusual for a cell to not produce output in some cases?

Reply:
> I think as long as Modin produces this output, we can leave it there. However, we should also consider not showing such low-level information, at least by default.

We can't remove a part of the cell output.

> Why? Is it unusual for a cell to not produce output in some cases?

I mean that some cells will be marked as [], i.e., not executed, but others will be marked as [<n>], i.e., executed, which looks inconsistent to me.

Reply:
We can do that in a separate issue when we have time. I believe we may even face other issues with interoperability of Modin and those packages. The notebooks were added a while ago and we don't test them.

Reply:
> We can do that in a separate issue when we have time. 

Should we not change them at all until then?

Reply:
I do not see an important reason of why we should not clear the cells output. We just say about interoperabilty (if it is there or not) in the notebooks. Maybe we should even recheck every notebooks on a regular basis to make sure the things are working as expected.